packages/backend/src/tokens/interstitial.ts

```diff
@@ -1,196 +0,0 @@
-import type { MultiDomainAndOrProxyPrimitives } from '@clerk/types';
-
-import { API_VERSION } from '../constants';
-// DO NOT CHANGE: Runtime needs to be imported as a default export so that we can stub its dependencies with Sinon.js
-// For more information refer to https://sinonjs.org/how-to/stub-dependency/
```

Comment:
❓ Don't we need to always return the `signed-out` state in place of the interstitial page? 

I understand we have 3 possible states:
- definitely `signed in`
- definitely `signed out`
- interstitial that will be now replaced by a `signed out` state